src/components/SectionTable/CourseInfoButton.js

```diff
@@ -11,7 +12,7 @@ const styles = {
     },
 };
 
-function CourseInfoButton({ classes, text, icon, redirectLink }) {
+function CourseInfoButton({ classes, text, icon, redirectLink, anlyticsAction, analyticsCategory }) {
```

Comment:
For components like this, I wonder if it would be better to pass in a function to log the analytics instead of the action and category?

Reply:
I don't think it makes enough difference to warrant changing it. If you have some good reasons then I can do it, but I just don't see why a function would be better.